dask_kubernetes/constants.py

```diff
@@ -0,0 +1 @@
+DASK_CONTAINER_NAME = "dask-worker"
```

Comment:
Given that this is specific only to `KubeCluster` and not `HelmCluster` or the new operator could we name this as something more specific.

Something like this

```suggestion
KUBECLUSTER_WORKER_CONTAINER_NAME = "dask-worker"
```

Reply:
Done